src/Raven.Server/Rachis/FollowerAmbassador.cs

```diff
@@ -317,11 +319,20 @@ private unsafe void Run()
                                         _engine.Log.Info($"{ToString()}: failure to append entries to {_tag} because: " + msg);
                                     }
 
-                                    throw new InvalidOperationException(msg);
+                                    RachisInvalidOperationException.Throw(msg);
                                 }
 
                                 if (aer.CurrentTerm != _term)
-                                    ThrowInvalidTermChanged(aer);
+                                {
+                                    var msg = $"The current engine term has changed " +
+                                              $"({aer.CurrentTerm:#,#;;0} -> {_term:#,#;;0}), " +
+                                              $"this ambassador term is no longer valid";
+                                    if (_engine.Log.IsInfoEnabled)
+                                    {
+                                        _engine.Log.Info($"{ToString()}: failure to append entries to {_tag} because: " + msg);
```

Comment:
Can do string interpolation there all the way

Reply:
done